src/Common/RandomNumber.cpp

```diff
@@ -0,0 +1,25 @@
+//=================================================================================================
+//  RandomNumber.cpp
+//  ..
+//
+//  This file is part of GANDALF :
+//  Graphical Astrophysics code for N-body Dynamics And Lagrangian Fluids
+//  https://github.com/gandalfcode/gandalf
+//  Contact : gandalfcode@gmail.com
+//
+//  Copyright (C) 2013  D. A. Hubber, G. Rosotti
+//
+//  GANDALF is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 2 of the License, or
+//  (at your option) any later version.
+//
+//  GANDALF is distributed in the hope that it will be useful, but
+//  WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//  General Public License (http://www.gnu.org/licenses) for more details.
+//=================================================================================================
+#include "RandomNumber.h"
+
+
+const FLOAT XorshiftRand::invrandmax = 1.0/1.84467440737095e19;
```

Comment:
I believe this is the 'correct' non c++11 solution. Adding a whole file for one line might seem like overkill, but at least this way interested parties would know where to look.